app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -332,6 +332,8 @@ public void onActivityResultSessionSafe(int requestCode, int resultCode, Intent
             } else if (requestCode == FormEntryConstants.INTENT_CALLOUT) {
                 processIntentResponse(intent, true);
                 Toast.makeText(this, Localization.get("intent.callout.cancelled"), Toast.LENGTH_SHORT).show();
+            } else if (requestCode == FormEntryConstants.VIEW_VIDEO_FULLSCREEN) {
+                Toast.makeText(this, Localization.get("media.invalid.video.path"), Toast.LENGTH_SHORT).show();
```

Comment:
This doesn't seem like the ideal validation place to me, we can potentially have other code path ways that can cancel the full screen activity. I think we should not even start the `FullscreenVideoViewActivity.kt` if the video uri is null or invalid and crash hard if we encounter the video path to be null inside the `FullscreenVideoViewActivity`

Reply:
Done. https://github.com/dimagi/commcare-android/pull/2747/commits/858864592f1a1190807e43ae7c128dcdea68c3d8

Reply:
we should remove the validation here now right ?

Reply:
Yes, it has been removed already, the code there is outdated. Can you double check?